src/Apprenticeships/SFA.DAS.Apprenticeships.Api/Extensions/NotificationMapperExtensions.cs

```diff
@@ -35,6 +36,13 @@ public static ChangeOfPriceApprovedCommand ToNotificationCommand(this PatchAppro
                 Approver = response.Approver
             };
         }
+        public static ChangeOfDateApprovedCommand ToNotificationCommand(this ApproveStartDateChangeRequest request, Guid apprenticeshipKey)
```

Comment:
Is it worth calling this ChangeOfStartDateApprovedCommand to be explicit and match the naming elsewhere

Reply:
Oh never mind just noticed you've called it this because the template is agnostic to start/end date